sycl/source/CMakeLists.txt

```diff
@@ -80,12 +80,17 @@ function(add_sycl_rt_library LIB_NAME)
       ${CMAKE_THREAD_LIBS_INIT}
     PUBLIC
       $<$<BOOL:${SYCL_BUILD_PI_CUDA}>:pi_cuda>
+      $<$<BOOL:${SYCL_BUILD_PI_ROCM}>:pi_rocm>
   )
 
   target_compile_definitions(${LIB_OBJ_NAME}
     PUBLIC
       $<$<BOOL:${SYCL_BUILD_PI_CUDA}>:USE_PI_CUDA>)
 
+  target_compile_definitions(${LIB_OBJ_NAME}
+    PUBLIC
+      $<$<BOOL:${SYCL_BUILD_PI_ROCM}>:USE_PI_ROCM>)
+
```

Comment:
```suggestion
```

I don't see why this is needed. Honestly, I don't understand why a similar CUDA piece of code exists. Looks like a bug in our build system.

Reply:
I added this line of code because I also saw CUDA code. I didn’t think too much, just followed the CUDA code.